pootle/apps/contact/forms.py

```diff
@@ -18,10 +18,10 @@
 
 class ContactForm(MathCaptchaForm, OriginalContactForm):
 
-    field_order = ['name', 'email', 'subject', 'body', 'captcha_answer',
+    field_order = ['name', 'email', 'summary', 'body', 'captcha_answer',
```

Comment:
Any name can be used for the field, except one that clashes with methods or fields coming from the  contact_form API. `subject` is a method in the API so we can use it for a field if we actually need the method as well, which we do, thus the renaming to "summary" which is a naming that was already used for it in other parts of the code. IMHO this makes it easier to spot the difference between the summary and the subject.